### DISCUSSION TITLE
plugin debug msg not show

### DISCUSSION THREAD

**Discussion by jyu99** [fwupd/fwupd#5951] [General]
I use this command, and I expect the pixart_rf plugin show some debug message e.g. fu-dump-raw or g_debug 

but it doesn't show it on console , shall I config other in fwupd?

<pre><font color="#4E9A06"><b>sdrf@OA-109048</b></font>:<font color="#3465A4"><b>~/fuwpd-20230628/fwupd/build</b></font>$ sudo ./src/fwupdtool get-devices --verbose --plugins=pixart_rf
<font color="#4E9A06">06:20:46.167 FuDebug              </font><font color="#3465A4">verbose to info (on console 1)</font>
<font color="#4E9A06">06:20:46.168 FuEngine             </font><font color="#3465A4">starting fwupd 1.9.3…</font>
Loading…                 [ -                                     ]<font color="#4E9A06">06:20:46.170 FuRemoteList         </font><font color="#3465A4">loading remote from /home/sdrf/fuwpd-20230628/fwupd-exe/etc/fwupd/remotes.d/dell-esrt.conf</font>
<font color="#4E9A06">06:20:46.170 FuRemoteList         </font><font color="#3465A4">loading remote from /home/sdrf/fuwpd-20230628/fwupd-exe/etc/fwupd/remotes.d/fwupd-tests.conf</font>
<font color="#4E9A06">06:20:46.170 FuRemoteList         </font><font color="#3465A4">loading remote from /home/sdrf/fuwpd-20230628/fwupd-exe/etc/fwupd/remotes.d/lvfs-testing.conf</font>
<font color="#4E9A06">06:20:46.170 FuRemoteList         </font><font color="#3465A4">loading remote from /home/sdrf/fuwpd-20230628/fwupd-exe/etc/fwupd/remotes.d/lvfs.conf</font>
<font color="#4E9A06">06:20:46.170 FuRemoteList         </font><font color="#3465A4">loading remote from /home/sdrf/fuwpd-20230628/fwupd-exe/etc/fwupd/remotes.d/vendor-directory.conf</font>
<font color="#4E9A06">06:20:46.171 FuRemoteList         </font><font color="#3465A4">loading remote from /home/sdrf/fuwpd-20230628/fwupd-exe/etc/fwupd/remotes.d/vendor.conf</font>
<font color="#4E9A06">06:20:46.171 FuRemoteList         </font><font color="#3465A4">enabled remotes: dell-esrt[1000], vendor-directory[1000], lvfs[0]</font>
Loading…                 [                                       ]<font color="#4E9A06">06:20:46.171 FuEngine             </font><font color="#3465A4">client certificate now exists: nothing to do</font>
Loading…                 [*                                      ]<font color="#4E9A06">06:20:46.173 FuQuirks             </font><font color="#3465A4">loading quirks from /home/sdrf/fuwpd-20230628/fwupd-exe/share/fwupd/quirks.d</font>
<font color="#4E9A06">06:20:46.173 FuQuirks             </font><font color="#3465A4">loading quirks from /home/sdrf/fuwpd-20230628/fwupd-exe/var/lib/fwupd/quirks.d</font>
Loading…                 [*                                      ]<font color="#4E9A06">06:20:46.175 FuContext            </font><font color="#3465A4">SMBIOS Manufacturer=Dell Inc.</font>
<font color="#4E9A06">06:20:46.175 FuContext            </font><font color="#3465A4">SMBIOS EnclosureKind=a</font>
<font color="#4E9A06">06:20:46.175 FuContext            </font><font color="#3465A4">SMBIOS Family=Latitude</font>
<font color="#4E9A06">06:20:46.175 FuContext            </font><font color="#3465A4">SMBIOS ProductName=Latitude 5300</font>
<font color="#4E9A06">06:20:46.175 FuContext            </font><font color="#3465A4">SMBIOS ProductSku=08B7</font>
<font color="#4E9A06">06:20:46.175 FuContext            </font><font color="#3465A4">SMBIOS BiosVendor=Dell Inc.</font>
<font color="#4E9A06">06:20:46.175 FuContext            </font><font color="#3465A4">SMBIOS BiosVersion=1.8.1</font>
<font color="#4E9A06">06:20:46.175 FuContext            </font><font color="#3465A4">SMBIOS BiosMajorRelease=01</font>
<font color="#4E9A06">06:20:46.175 FuContext            </font><font color="#3465A4">SMBIOS BiosMinorRelease=08</font>
<font color="#4E9A06">06:20:46.175 FuContext            </font><font color="#3465A4">SMBIOS FirmwareMajorRelease=ff</font>
<font color="#4E9A06">06:20:46.175 FuContext            </font><font color="#3465A4">SMBIOS FirmwareMinorRelease=ff</font>
<font color="#4E9A06">06:20:46.175 FuContext            </font><font color="#3465A4">SMBIOS BaseboardManufacturer=Dell Inc.</font>
<font color="#4E9A06">06:20:46.175 FuContext            </font><font color="#3465A4">SMBIOS BaseboardProduct=0932VT</font>
<font color="#4E9A06">06:20:46.175 FuContext            </font><font color="#3465A4">failed to load fdt: cannot find /sys/firmware/fdt or override /home/sdrf/fuwpd-20230628/fwupd-exe/var/lib/fwupd/system.dtb</font>
<font color="#4E9A06">06:20:46.185 FuContext            </font><font color="#3465A4">added udev subsystem watch of firmware-attributes</font>
<font color="#4E9A06">06:20:46.900 FuBiosSettings       </font><font color="#3465A4">loaded 123 BIOS settings</font>
Loading…                 [**                                     ]<font color="#4E9A06">06:20:46.900 FuEngine             </font><font color="#3465A4">loading metadata for remote &apos;vendor-directory&apos;</font>
<font color="#4E9A06">06:20:46.900 FuEngine             </font><font color="#3465A4">ignoring: /home/sdrf/fuwpd-20230628/fwupd-exe/share/fwupd/remotes.d/vendor/firmware/README.md</font>
<font color="#4E9A06">06:20:46.900 FuEngine             </font><font color="#3465A4">ignoring: Error opening directory “/home/sdrf/fuwpd-20230628/fwupd-exe/var/lib/fwupd/local.d”: No such file or directory</font>
<font color="#4E9A06">06:20:46.900 FuEngine             </font><font color="#3465A4">ignoring: Error opening directory “/home/sdrf/fuwpd-20230628/fwupd-exe/share/fwupd/local.d”: No such file or directory</font>
<font color="#4E9A06">06:20:46.900 FuEngine             </font><font color="#3465A4">1 components now in silo</font>
Loading…                 [***                                    ]<font color="#4E9A06">06:20:46.911 FuContext            </font><font color="#3465A4">added udev subsystem watch of hidraw</font>
<font color="#4E9A06">06:20:46.911 FuEngine             </font><font color="#3465A4">plugins disabled: uefi_capsule, acpi_dmar, acpi_facp, acpi_ivrs, acpi_phat, amd_pmc, amd_gpu, analogix, android_boot, ata, aver_hid, bcm57xx, bios, ccgx, ccgx_dmc, cfu, ch341a, ch347, colorhug, corsair, cpu, cros_ec, dell, dell_dock, dell_esrt, dfu, dfu_csr, ebitdo, elantp, elanfp, emmc, ep963x, fastboot, focalfp, fpc, fresco_pd, goodixmoc, goodixtp, hailuck, igsc, intel_me, intel_usb4, iommu, jabra, jabra_gnp, lenovo_thinklmi, linux_lockdown, linux_sleep, linux_swap, linux_tainted, logind, logitech_hidpp, logitech_bulkcontroller, logitech_scribe, logitech_tap, msr, mtd, nitrokey, nordic_hid, nvme, optionrom, parade_lspcon, pci_bcr, pci_mei, pci_psp, powerd, qsi_dock, realtek_mst, redfish, rts54hid, rts54hub, steelseries, scsi, superio, synaptics_cape, synaptics_cxaudio, synaptics_mst, synaptics_prometheus, synaptics_rmi, system76_launch, test, test_ble, thelio_io, thunderbolt, ti_tps6598x, tpm, uefi_dbx, uefi_pk, uefi_recovery, uf2, upower, usi_dock, vbe, vli, wacom_raw, wacom_usb, wistron_dock</font>
Loading…                 [***                                    ]<font color="#4E9A06">06:20:46.919 FuContext            </font><font color="#3465A4">battery threshold now 10</font>
Loading…                 [*****                                  ]<font color="#4E9A06">06:20:46.919 FuEngine             </font><font color="#3465A4">using plugins: pixart_rf</font>
Loading…                 [*************************              ]<font color="#4E9A06">06:20:47.172 FuDevice             </font><font color="#CC0000">1.0 is not a valid triplet (guessed pair)</font>
Loading…                 [*************************              ]<font color="#4E9A06">06:20:47.212 FuDeviceList         </font><font color="#3465A4">::added d93e87906ab431584d4def122e5900deed61d143 [M2862]</font>
Loading…                 [**************************             ]<font color="#4E9A06">06:20:47.220 FuDevice             </font><font color="#3465A4">failed on try 1 of 10: ioctl error: Input/output error [5]</font>
<font color="#4E9A06">06:20:47.270 FuDevice             </font><font color="#3465A4">failed on try 2 of 10: ioctl error: Input/output error [5]</font>
<font color="#4E9A06">06:20:47.320 FuDevice             </font><font color="#3465A4">failed on try 3 of 10: ioctl error: Input/output error [5]</font>
<font color="#4E9A06">06:20:47.371 FuDevice             </font><font color="#3465A4">failed on try 4 of 10: ioctl error: Input/output error [5]</font>
<font color="#4E9A06">06:20:47.421 FuDevice             </font><font color="#3465A4">failed on try 5 of 10: ioctl error: Input/output error [5]</font>
<font color="#4E9A06">06:20:47.472 FuDevice             </font><font color="#3465A4">failed on try 6 of 10: ioctl error: Input/output error [5]</font>
<font color="#4E9A06">06:20:47.522 FuDevice             </font><font color="#3465A4">failed on try 7 of 10: ioctl error: Input/output error [5]</font>
<font color="#4E9A06">06:20:47.572 FuDevice             </font><font color="#3465A4">failed on try 8 of 10: ioctl error: Input/output error [5]</font>
<font color="#4E9A06">06:20:47.623 FuDevice             </font><font color="#3465A4">failed on try 9 of 10: ioctl error: Input/output error [5]</font>
<font color="#4E9A06">06:20:47.673 FuEngine             </font><font color="#CC0000">failed to add device /sys/devices/virtual/misc/uhid/0005:03F0:114C.0015/hidraw/hidraw7: failed to OTA check retransmit: failed after 10 retries: ioctl error: Input/output error [5]</font>
Loading…                 [***************************            ]<font color="#4E9A06">06:20:47.673 FuBackend            </font><font color="#3465A4">adding suitable BlueZ device: /org/bluez/hci0/dev_C9_5C_A8_A5_EB_69</font>
<font color="#4E9A06">06:20:47.673 FuBackend            </font><font color="#3465A4">adding suitable BlueZ device: /org/bluez/hci0/dev_DD_BC_99_BA_1A_CE</font>
Loading…                 [************************************** ]<font color="#4E9A06">06:20:47.678 FuEngine             </font><font color="#3465A4">FuUsbBackend:</font>
<font color="#3465A4">  Name:                 usb</font>
<font color="#3465A4">  Enabled:              true</font>
<font color="#3465A4">  DoneSetup:            true</font>
<font color="#3465A4">  CanInvalidate:        false</font>
<font color="#3465A4">FuUdevBackend:</font>
<font color="#3465A4">  Name:                 udev</font>
<font color="#3465A4">  Enabled:              true</font>
<font color="#3465A4">  DoneSetup:            true</font>
<font color="#3465A4">  CanInvalidate:        false</font>
<font color="#3465A4">FuBluezBackend:</font>
<font color="#3465A4">  Name:                 bluez</font>
<font color="#3465A4">  Enabled:              true</font>
<font color="#3465A4">  DoneSetup:            true</font>
<font color="#3465A4">  CanInvalidate:        false</font>
<font color="#3465A4">FuPxiPlugin:</font>
<font color="#3465A4">  Name:                 pixart_rf</font>

<font color="#4E9A06">06:20:47.682 FuEngine             </font><font color="#3465A4">resetting update motd timeout</font>

<font color="#CC0000"><b>WARNING</b></font>: This package has not been validated, it may not work properly.
Dell Inc. Latitude 5300
│
└─M2862:
      Device ID:          d93e87906ab431584d4def122e5900deed61d143
      Current version:    1.0
      Vendor:             USB:0x093A|HIDRAW:0x03F0
      GUIDs:              d5f1bb18-3e28-574d-85ea-ac4aec52dc45 ← HIDRAW\VEN_03F0&amp;DEV_114C
                          c7a34200-a201-5158-945d-cc1293000ac7 ← HIDRAW\VEN_03F0&amp;DEV_114C&amp;NAME_M2862
                          58cc44ce-6bed-520b-8a96-996a21ac40a1 ← HIDRAW\VEN_03F0&amp;DEV_114C&amp;MODEL_M2862
      Device Flags:       • Updatable
                          • Unsigned Payload
</pre>

**Comment by hughsie** [comment]
Add an extra `--verbose` or use `-vv` instead to increase the log level.
